test/intern_test.c

```diff
@@ -83,7 +101,13 @@ void load_words(void) {
   while(fgets(buff, sizeof(buff), fp)) {
     if(strlen(buff) < 1) continue;
     if(buff[strlen(buff)-1] == '\n') buff[strlen(buff)-1] = '\0';
-    words[word_cnt++] = strdup(buff);
+    int upsize = mtev_rand() % 4096;
+    if(upsize < strlen(buff)) upsize = strlen(buff);
+    char *str = malloc(upsize+1);
```

Comment:
Where do we call `free(str)`?

Reply:
It is not freed, it's building a table that's used until exit.